segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/ecs/ECSSegmentHandle.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.ecs;
+
+import io.pravega.segmentstore.storage.SegmentHandle;
+
+public class ECSSegmentHandle implements SegmentHandle {
```

Comment:
Understood that no one else other than ECS uses Extended S3 at the moment, but for the sake of consistency, let's use `ExtS3` rather than `ECS` across the code.